spec/mailers/previews/defect_report_mailer_preview.rb

```diff
@@ -11,11 +11,9 @@ def notify_for_category
     generic_item.content_blocks = [OpenStruct.new(body: "Test mit mehr Worten", title: "Test")]
     generic_item.external_reference = OpenStruct.new(unique_id: "1234567890")
 
-    StaticContent.first_or_create(
-      name: "defectreport-notify-for-category",
-      data_type: "json",
-      content: "{\"app_name\":\"Test-App\",\"mail_footer\":[\"Zeile 1\", \"\", \"Zeile 2\"]}"
-    )
+    StaticContent.find_or_create_by(name: 'defectreport-notify-for-category', data_type: "json" ) do |s|
```

Comment:
please do not mix quotes and use double quotes everywhere. the `name: ...` is single quoted in every file changed